apps/studio/prisma/seed.ts

```diff
@@ -58,10 +86,13 @@ async function main() {
       config: {
         theme: 'isomer-next',
         sitemap: {
-          siblingTitles: [],
-          childrenTitles: [],
-          parentTitle: '',
-        },
+          title: 'Home',
+          permalink: '/',
+          children: [],
+          layout: 'content',
+          summary: 'something',
+          lastModified: '',
+        } satisfies IsomerSitemap,
```

Comment:
NOTE: actually this should be dynamically generated. this is omitted at present.

Reply:
Yeah hmm actually we shouldn't be storing the sitemap inside the DB since it should be derived when rendering the preview, but yeah we can remove this in a subsequent PR!